pkg/compare/path.go

```diff
@@ -51,14 +51,28 @@ func (p Path) Pop() {
 	}
 }
 
-// Contains returns true if the supplied string appears within the Path
+//	Contains returns true if the supplied string, delimited on ".", matches
+//	p.parts up to the length of the supplied string.
```

Comment:
femto-nit: not sure if it's your IDE or editor changing the above, but comments should have a single space (not `\t`) after the `//` and before the first letter of the comment...

Reply:
Got it, will keep in mind for next time